src/components/Explorer/Rdrp/LinkButtons.jsx

```diff
@@ -35,6 +38,20 @@ function SequenceLinkButtons({ sequence_accession }) {
 }
 
 function RunLinkButtons({ run_id }) {
+    const baseUrl = window.location.origin
+    const [microAvailable, setMicroAvailable] = React.useState(false)
+    const [ntAvailable, setNtAvailable] = React.useState(false)
+
+    React.useEffect(() => {
+        async function setIndexButtons() {
+            const responseData = await getAnalysisIndex(run_id)
+            setMicroAvailable(responseData.analysis_index.micro)
```

Comment:
I don't like the intermediate `analysis_index` here. Suggest we change the API to return without nesting:

```json
{
    "run_id": "ERR2756788",
    "srarun": true,
    "nsra": true,
    "psra": true,
    "rsra": true,
    "assembly": true,
    "micro": true,
    "geo": false
}
```

then we can use this:

```javascript
const analysisIndex = await getAnalysisIndex(run_id)
setMicroAvailable(analysisIndex.micro)
setNtAvailable(analysisIndex.nsra)
```

Reply:
Fixed it by changing the function return to `analysis_index`

Reply:
```
export const getAnalysisIndex = async (run_id) => {
    const analysisUrl = `${baseUrl}/index/run=${run_id}`
    const response = await axios.get(analysisUrl)
    return response.data.analysis_index
}
```